doc/application-management/application-management.md

```diff
@@ -279,6 +285,21 @@ rbac:
   #   resources: []
 ```
 
+You can view the configuration of an application using the catalog built into the CLI. This can be done without a running cluster.
+```
+$ ocne application template --name grafana --catalog embedded | head
```

Comment:
may be `less` instead of `head`?

Reply:
I was copying from the example above.  I think head was chosen to limit the sample output.  I will switch to less so that the full output becomes visible when using the example.

Reply:
I just pushed that change